scripts/pre_commit_linter.py

```diff
@@ -169,7 +169,7 @@
 
 EXCLUDED_PATHS = (
     'third_party/*', 'build/*', '.git/*', '*.pyc', 'CHANGELOG',
-    'integrations/*', 'integrations_dev/*', '*.svg',
+    'integrations/*', 'integrations_dev/*', '*.svg', '*.gif',
```

Comment:
This is added after consultation with Apurv?

Reply:
Yes!

Reply:
(Confirming the same!)